_rules/document-title-is-descriptive.md

```diff
@@ -0,0 +1,200 @@
+- This rule is testing [Technique G88: Providing descriptive titles for Web pages](https://www.w3.org/WAI/WCAG21/Techniques/general/G88)
+- A prerequisite for this rule is that [Technique H25: Providing a title using the title element](https://www.w3.org/WAI/WCAG21/Techniques/html/H25) is used for living up to 2.4.2 Page Titled
+
+## Test Cases
+
+### Passed
+
+#### Passed example 1
+
+The `<title>` element describes the content of the document.
+
+```html
+<html>
+  <head>
+    <title>Clementine harvesting season</title>
+  </head>
+  <body>
+    <p>Clementines will be ready to harvest from late October through February.</p>
+  </body>
+</html>
+```
+
+#### Passed example 2
+
+Two `<title>` elements where the first one describes the content of the document.
+
+```html
+<html>
+  <head>
+    <title>Clementine harvesting season</title>
+    <title>Apple harvesting season</title>
+  </head>
+  <body>
+    <p>Clementines will be ready to harvest from late October through February.</p>
+  </body>
+</html>
+```
+
+#### Passed example 3
+
+Even though the descriptive `<title>` element is not placed within the `<head>` element that is the context the element can be used in [according to the HTML specification](https://www.w3.org/TR/html/document-metadata.html#the-title-element), the rule still passes, since the browser fixes it, and it doesn't cause any known accessibility issues.
+
+```html
+<html>
+  <head>
+  </head>
+  <body>
+    <title>Clementine harvesting season</title>
+    <p>Clementines will be ready to harvest from late October through February.</p>
+  </body>
+</html>
+```
+
+### Failed
+
+#### Failed example 1
+
+`<title>` is not descriptive of the content of the document.
+
+```html
+<html>
+  <head>
+    <title>Apple harvesting season</title>
+  </head>
+  <body>
+    <p>Clementines will be ready to harvest from late October through February.</p>
+  </body>
+</html>
+```
+
+### Failed example 2
+
+Even though a correct `<title>` element is put in the `<head>` of the document, this rule only looks at the first `<title>` element.
+
+```html
+<html>
+  <head>
+    <title>Apple harvesting season</title>
+    <title>Clementine harvesting season</title>
+  </head>
+  <body>
+    <p>Clementines will be ready to harvest from late October through February.</p>
+  </body>
+</html>
+```
+
+
+### Inapplicable
+
+#### Inapplicable example 1
+
+No `<title>` element present in document.
+
+```html
+<html>
+  <head>
+  </head>
+  <body>
+    <p>Clementines will be ready to harvest from late October through February.</p>
+  </body>
+</html>
+```
+
+#### Inapplicable example 2
+
+`<title>` element present in document, but is empty.
+
+```html
+<html>
+  <head>
+    <title></title>
+  </head>
+  <body>
+    <p>Clementines will be ready to harvest from late October through February.</p>
+  </body>
+</html>
+```
+
+#### Inapplicable example 3
+
+`<title>` element present in document, but does not meet the definition for [non-empty text string](#non-empty).
```

Comment:
You have engaged a work on the "non-empty" glossary definition (#430), so, if I understand well, the "non-empty text string" here is not a definition in autowcag anymore ? 

And, since the applicability become "contains text node", is it right to consider this example a passed one ?

Reply:
Great catch, @audreymaniez!
I have moved the test case to Failed (and changed the wording to not use the non-empty definition) and added a similar test case for Passed.